pkg/sfu/downtrack.go

```diff
@@ -578,6 +585,7 @@ func (d *DownTrack) CloseWithFlush(flush bool) {
 
 		d.callbacksQueue.Stop()
 		d.keyFrameRequestGeneration.Inc()
+		close(d.qc)
```

Comment:
@cnderrauber Any reason you see why the generation counter increment in the previous line does not work? That counter is exactly for that. Do you have a set up where I can reproduce this? I would like to understand why that is not working and if it is an indication of some deeper problem. I am thinking `maybeStartKeyFrameRequester` should not start a new one when down track is closed or not bound. Would prefer that rather than introducing another channel.

Reply:
the counter inc not work is because it start after close been called.

Reply:
@boks1971 changed to bound check. If prevent maybeStartKeyFrameRequester to start, might import a lock or a similar thing?

Reply:
It should not get a lock. But, this change should work. I am also making a small change to refactor a bit. I will commit to your branch. Can you please wait till then?

Reply:
Not worry, I can test on my side

Reply:
Thank you @cnderrauber . This is the commit - https://github.com/livekit/livekit-server/pull/527/commits/e4209644381ba5433c8537cf5b289a97951a04fc. I could not reproduce the original problem in my local set up. If you can try this and it works, please go ahead.

Reply:
verified it can work.